convert2rhel/main.py

```diff
@@ -85,7 +85,11 @@ def main():
         systeminfo.system_info.resolve_system_info()
         breadcrumbs.breadcrumbs.collect_early_data()
 
+        loggerinst.task("Prepare: Clean yum cache metadata")
+        pkghandler.clean_yum_metadata()
+
         # check the system prior the conversion (possible inhibit)
+        loggerinst.task("Prepare: Perform basic system checks")
         checks.perform_pre_checks()
```

Comment:
Codecov is mad because there is no unit test for this specific function, should we try to implement a basic mock test of this function?